test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -195,8 +242,33 @@ func (s *Suite) Define() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
-		It("should issue a certificate that defines a commonName and sets a duration", func() {
-			s.checkFeatures(DurationFeature)
+		It("should issue a certificate that defines a URI Name and URI SAN", func() {
+			s.checkFeatures(URISANsFeature)
+			s.checkFeatures(CommonNameFeature)
+
+			testCertificate := &cmapi.Certificate{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "testcert",
+					Namespace: f.Namespace.Name,
+				},
+				Spec: cmapi.CertificateSpec{
+					SecretName: "testcert-tls",
+					CommonName: "test-common-name",
+					URISANs:    []string{"spiffe://cluster.local/ns/sandbox/sa/foo"},
+					IssuerRef:  issuerRef,
+				},
+			}
+			By("Creating a Certificate")
+			err := f.CRClient.Create(ctx, testCertificate)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Waiting for the Certificate to be issued...")
+			err = f.Helper().WaitCertificateIssuedValid(f.Namespace.Name, "testcert", time.Minute*5)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should issue a certificate that defines a 2 distinct DNS Name with one copied to the Common Name", func() {
```

Comment:
Do we have a test the sets:

```
commonName: something.com
dnsNames:
- somethingelse.com
```

? and do you think it's a case worth checking?

Reply:
Can't hurt (famous last words)